src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -119,6 +119,10 @@ public FeatureOnOffOptions()
             FeatureName, "AutomaticallyCompleteStatementOnSemicolon", defaultValue: true,
             storageLocation: new RoamingProfileStorageLocation("TextEditor.AutomaticallyCompleteStatementOnSemicolon"));
 
+        public static readonly PerLanguageOption2<bool> AutomaticallyFixStringContentsOnPaste = new(
```

Comment:
\<Youssef\>AutomationObject\</Youssef\>

Reply:
it's there :)

![image](https://user-images.githubusercontent.com/4564579/158307048-c0b1add2-c8dc-4779-a36f-723b8022fe77.png)

Reply:
🤦‍♂️
I blame daylight savings

Reply:
> <Youssef>AutomationObject</Youssef>

One day I'll get to writing a source generator for it 😄